doc/installation.rst

```diff
@@ -264,6 +265,25 @@ In summary:
 
    This installs all of Spyder's dependencies into the environment along with the stable/packaged version of Spyder, and then removes Spyder itself.
 
+   .. note::
+
+      Following the separation in v3.3 of Spyder's console code into its own package, ``spyder-kernels``, you'll need to have the corresponding version of it available—``0.x`` for Spyder 3 (``3.x`` branch), and ``1.x`` for Spyder 4 (``master`` branch).
+      The above procedure will install the ``0.x`` version; to test the ``master`` branch (Spyder 4), you'll need to install the corresponding ``1.x`` version of ``spyder-kernels``.
+      This can be done via two methods: installing the ``1.x`` version via ``conda``:
+
+      .. code-block:: bash
+
+         conda install -c conda-forge spyder-kernels=1
+
+      or ``pip``:
+
+      .. code-block:: bash
+
+         pip install --no-deps -U spyder-kernels==1.*
```

Comment:
Since we're going to use conda to install `spyder-kernels`, this could be simpler:

    pip install spyder-kernels==1.*

Reply:
Okay, gotcha. I figured that might be but wasn't 100% sure. Done.